_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Image accessible name is placeholder text
```

Comment:
Hmm. I think most rule titles state the success condition rather than the failure condition - for example, "Image accessible name is suitable placeholder text".